### ISSUE TITLE
Holistic example code is not working

### ISSUE THREAD

**Issue by AlphaNote** [google/mediapipe#3681]
**System information** (Please provide as much relevant information as possible)
~~- Have I written custom code (as opposed to using a stock example script provided in Mediapipe):~~
~~- OS Platform and Distribution (e.g., Linux Ubuntu 16.04, Android 11, iOS 14.4):~~
- [MediaPipe version](https://github.com/google/mediapipe/releases): mediapipe-0.8.10.1-cp37-cp37m-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
~~- Bazel version:~~
- Solution (e.g. FaceMesh, Pose, Holistic): Holistic
- Programming Language and version ( e.g. C++, Python, Java): Python

**Describe the expected behavior:**

It doesn't work because it doesn't receive a "pose_landmark_heavy.tflite".

Perhaps the file was deleted due to recent work.

**Standalone code you may have used to try to get what you need :**

If there is a problem, provide a reproducible test case that is the bare minimum necessary to generate the problem. If possible, please share a link to Colab/repo link /any notebook:

[https://mediapipe.page.link/holistic_py_colab](https://mediapipe.page.link/holistic_py_colab)
[https://colab.research.google.com/drive/16UOYQ9hPM6L5tkq7oQBl1ULJ8xuK5Lae?usp=sharing#scrollTo=BAivyQ_xOtFp](https://colab.research.google.com/drive/16UOYQ9hPM6L5tkq7oQBl1ULJ8xuK5Lae?usp=sharing#scrollTo=BAivyQ_xOtFp)

**Other info / Complete Logs :**

```
Downloading model to /usr/local/lib/python3.7/dist-packages/mediapipe/modules/pose_landmark/pose_landmark_heavy.tflite
---------------------------------------------------------------------------
HTTPError                                 Traceback (most recent call last)
[<ipython-input-9-4b9d4f973a30>](https://localhost:8080/#) in <module>
      1 # Run MediaPipe Holistic and draw pose landmarks.
      2 with mp_holistic.Holistic(
----> 3     static_image_mode=True, min_detection_confidence=0.5, model_complexity=2) as holistic:
      4   for name, image in images.items():
      5     # Convert the BGR image to RGB and process it with MediaPipe Pose.

8 frames
[/usr/lib/python3.7/urllib/request.py](https://localhost:8080/#) in http_error_default(self, req, fp, code, msg, hdrs)
    647 class HTTPDefaultErrorHandler(BaseHandler):
    648     def http_error_default(self, req, fp, code, msg, hdrs):
--> 649         raise HTTPError(req.full_url, code, msg, hdrs, fp)
    650 
    651 class HTTPRedirectHandler(BaseHandler):

HTTPError: HTTP Error 404: Not Found
```

**Comment by kuaashish** [comment]
Hi @AlphaNote, Please pull the latest update from the `master` branch it must work now. Refer these two issues for better understading #3674 and [#3675](https://github.com/google/mediapipe/issues/3674). Thank you!

**Comment by kuaashish** [comment]
Hi @AlphaNote, Could you please confirm this is still an issue or resolved as per the [above](https://github.com/google/mediapipe/issues/3681#issuecomment-1240275251) comment. Thank you!

**Comment by AlphaNote** [comment]
Hi @kuaashish, Thank you for your comment. 
I solved the problem by referring to #3674 and #3675. And I checked that the version has been upgraded to 0.8.11 in pip.

`mediapipe-0.8.11-cp37-cp37m-manylinux_2_17_x86_64.manylinux2014_x86_64.whl`
